src/index.js

```diff
@@ -1 +1,53 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import displayScores from './modules/api.js';
+
+const refreshButton = document.querySelector('#refresh');
+const form = document.querySelector('.inputPlayer');
+const message = document.querySelector('#message');
+const key = 'Fre06jK3qCMjBipi1fVQ';
+// const key = process.env.API_KEY;
```

Comment:
- [optional] kindly remove the commented line for cleaner code(YAGNI). please when the reviewer talks of removing commented code you check and do so for the whole project not just that particular line. nice work